Src/AutoFixture/InstanceGenerator.cs

```diff
@@ -1,164 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Reflection;
-
-namespace Ploeh.AutoFixture
-{
-    /// <summary>
-    /// Default reusable implementation of <see cref="IInstanceGenerator" />
-    /// </summary>
-    public class InstanceGenerator : IInstanceGenerator

```

Comment:
Like `OmitRecursionGuard`, this seems to be an orphaned file. As far as I can tell, the file was renamed from within Visual Studio in 0abcc8afd86a378b9b919ca0e8a4de849e3b9401, but the source control system wasn't told about the rename, which caused it to remain in the repository. Clearly, in early 2010, I was still struggling with the version control system in use back then (most likely Mercurial).

Since this file is orphaned, I think it should be deleted from *master* as well, so this needs to happen in a separate pull request.

Again, I'll be happy to do this myself, but equally happy to let someone else do it, in order to get 'contribution credit'.